ros/api/v1/hosts.py

```diff
@@ -41,7 +41,9 @@
     '8.2': {"name": "RHEL", "major": 8, "minor": 2},
     '8.3': {"name": "RHEL", "major": 8, "minor": 3},
     '8.4': {"name": "RHEL", "major": 8, "minor": 4},
-    '8.5': {"name": "RHEL", "major": 8, "minor": 5}
+    '8.5': {"name": "RHEL", "major": 8, "minor": 5},
+    '8.6': {"name": "RHEL", "major": 8, "minor": 6},
+    '9.0': {"name": "RHEL", "major": 8, "minor": 0},
```

Comment:
```suggestion
    '9.0': {"name": "RHEL", "major": 9, "minor": 0},
```

Reply:
@upadhyeammit Oops! Thanks Amit

Reply:
I think we don't need the `,` for last row? and I missed this in suggestion :| 
~~~
'9.0': {"name": "RHEL", "major": 9, "minor": 0}
~~~

Reply:
I prefer to add trailing commas such that the next addition is easier and also avoids a syntax error.

Reference: https://peps.python.org/pep-0008/#when-to-use-trailing-commas

Open to suggestions :slightly_smiling_face:

Reply:
then it makes sense!